recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "6.1.0" %}
+{% set version = "6.2.0" %}
 
 package:
   name: nbconvert
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/n/nbconvert/nbconvert-{{ version }}.tar.gz
-  sha256: d22a8ff202644d31db254d24d52c3a96c82156623fcd7c7f987bba2612303ec9
+  sha256: 16ceecd0afaa8fd26c245fa32e2c52066c02f13aa73387fffafd84750baea863
```

Comment:
`sha256` for 6.3.0 is`5e77d6203854944520105e38f2563a813a4a3708e8563aa598928a3b5ee1081a`

Reply:
The `sha256` value was updated:
https://github.com/AnacondaRecipes/nbconvert-feedstock/blob/update_to_6.2.0/recipe/meta.yaml#L9